### PR TITLE
Updated cross refs to "Supported AWS Regions" section

### PR DESCRIPTION
**Pull request by MarciaRieferJohnston**
Update the deployment procedures' cross refs to the newly genericized "Supported AWS Regions" section.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: deployment_steps_eks_module.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ NOTE: This step is only required if you launch this Quick Start into an existing
 
 . Sign in to your AWS account at https://aws.amazon.com[https://aws.amazon.com^] with an IAM user role that has the necessary permissions. For details, see link:#_planning_the_deployment[Planning the deployment], earlier in this guide.
 . Launch the https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/create/template?stackName=Amazon-EKS&templateURL=https://aws-quickstart.s3.us-east-1.amazonaws.com/quickstart-amazon-eks/templates/amazon-eks-entrypoint-existing-cluster.template.yaml[cluster preparation template^].
-. The template launches in the US East (Ohio) Region by default. To change the Region, choose another Region from the list in the upper-right corner of the navigation bar.
+. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where you build the network infrastructure. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```
Comment (jaymccon) [inline deployment_steps_eks_module.adoc:8]: same as above

File: deployment_steps_eks_module.adoc [hunk 2]
```diff
@@ -40,7 +40,7 @@ New clusters take about 1.5 hours to deploy. Existing clusters take about {deplo
 +
 WARNING: If you deploy {partner-product-short-name} into an existing VPC, ensure that any private subnets have https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html[NAT gateways^] in their route tables to allow the Quick Start to download packages and software. Also, ensure that the domain name in the DHCP options is configured. For more information, see http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_DHCP_Options.html[DHCP options sets^].
 +
-. Check the AWS Region that’s displayed in the upper-right corner of the navigation bar, and change it if necessary. This is where you build the network infrastructure. The template launches in the {default_deployment_region} Region by default.
+. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where you build the network infrastructure. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```
Comment (jaymccon) [inline deployment_steps_eks_module.adoc:43]: same as above

File: .specific/deploy_steps.adoc [hunk 3]
```diff
@@ -35,6 +35,6 @@ Each deployment takes about {deployment_time} to complete.
 ^|http://qs_template_permalink[View template^]
 |===
 +
-. Check the AWS Region that’s displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where you build the network infrastructure. The template is launched in the {default_deployment_region} Region by default. For other choices, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
+. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where you build the network infrastructure. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```
Comment (jaymccon) [inline .specific/deploy_steps.adoc:38]: I realize this text was already present in the guide, but just noticed this. Can we change "the network infrastructure" to "the infrastructure", quick starts generally deploys more than just network infrastructure, so current statement is confusing.

@MarciaRieferJohnston you need to show me sometime how you do that propose change thing...

**Review by jaymccon — CHANGES_REQUESTED** [review]

File: deployment_steps_eks_module.adoc (hunk 1 above)
Comment (MarciaRieferJohnston) [reply deployment_steps_eks_module.adoc:8]: @jaymccon I will remove "network." 

As for that propose change thing, isn't that slick? I figured all the SAs would know about this! I learned about it during the preconference Writing Day at Write the Docs a few months ago. (Always an excellent conference for open-source developers and tech writers, aka "documentarians.") Check out step 6 here: https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request

After you suggest your edit, you click the **Add single comment** button and commit the change.

Comment (MarciaRieferJohnston) [reply deployment_steps_eks_module.adoc:8]:
```suggestion
. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where you build the infrastructure. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```

File: .specific/deploy_steps.adoc (hunk 3 above)
Comment (MarciaRieferJohnston) [reply .specific/deploy_steps.adoc:38]:
```suggestion
. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where you build the infrastructure. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```

File: deployment_steps_eks_module.adoc (hunk 2 above)
Comment (MarciaRieferJohnston) [reply deployment_steps_eks_module.adoc:43]:
```suggestion
. Check the AWS Region that's displayed in the upper-right corner of the navigation bar, and change it if necessary. This Region is where you build the infrastructure. The template is launched in the {default_deployment_region} Region by default. For more information, see link:#_supported_aws_regions[Supported AWS Regions] earlier in this guide.
```

**Review by jaymccon — APPROVED** [review]

**Review by censullo — APPROVED** [review]

**Pull request merged**